packages/realm/src/flags.ts

```diff
@@ -26,4 +26,9 @@ export const flags = {
    * Helps finding places where the app is depending on the now deprecated way of using the package.
    */
   THROW_ON_GLOBAL_REALM: false,
+  /**
+   * When enabled, calling `Realm.clearTestState` will be callable.
+   * This is disabled by default, mainly because the data-structures needed to support this, introduce minor memory leaks and are not intended for production use.
```

Comment:
```suggestion
   * This is disabled by default, mainly because the data-structures needed to support this, introduce minor memory leaks if clearTestState() is not called regularly and are not intended for production use.
```

Reply:
I added this.